src/Components/Bookitem.js

```diff
@@ -1,17 +1,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBoook } from '../redux/books/books';
 
 const Bookitems = (props) => {
-  const { title, author } = props;
+  const {
+    title, author, id,
+  } = props;
+  const dispatch = useDispatch();
+  const handleRemoveClick = () => {
+    dispatch(removeBoook(id));
+  };
   return (
     <div className="book-items">
       <li className="bk">{title}</li>
       <li className="bk-2">{author}</li>
+      <button type="button" onClick={handleRemoveClick}>Remove</button>
     </div>
```

Comment:
- [ ] **[OPTIONAL]** Also, remember that `<li>` are elements reserved for list items, which means that under an element different than a list, they are invalid markups. Please consider using another tag for these elements! I suggest any `<hx>` one, such as `<h4>`. 🤖 🔧